injected/src/utils.js

```diff
@@ -418,7 +406,7 @@ export class DDGProxy {
             }
             // The normal return value
             if (isExempt) {
-                return DDGReflect.apply(...args);
+                return DDGReflect.apply(args[0], args[1], args[2]);
```

Comment:
What's the rationale for this? (if none I wouldn't include it, if there is why not the same for the proxyObject version?)

Reply:
There was a weird Typescript lint error here

Reply:
Weird!